docs/README.md

```diff
@@ -249,7 +249,7 @@ cat $HOME/.docker/config.json | docker secret create registry-secret -
 Create of-builder and of-buildkit:
 
 ```
-./of-builder/deploy_swarm.sh
```

Comment:
when I ran this command I got the following output:

```
./of-builder/deploy_swarm.sh
Error: No such container: of-buildkit
Unable to find image 'alexellis2/buildkit:2018-04-17' locally
2018-04-17: Pulling from alexellis2/buildkit
ff3a5c916c92: Already exists
41b99add21a2: Pull complete
75b79d1d2c70: Pull complete
c5da8e9d31f3: Pull complete
15eb2f52b88e: Pull complete
Digest: sha256:a05aad0a2344ee7b643a2ecbe5073c8636cee73b3c2f36ce13d8afe262311188
Status: Downloaded newer image for alexellis2/buildkit:2018-04-17
1330cd74d587ab40175f19944fb5bc37fa3567e33be70f46ae9d61bad1b49295
docker: Error response from daemon: linux runtime spec devices: lstat /dev/bpf23: input/output error.
o3dgh1arm1ki9uhlkbpwvyry9
```

Nothing in docker logs

Reply:
installing docker edge fixed this problem, worth having a trouble shooting section at the end?